uyuni/base/uyuni-base.spec

```diff
@@ -89,11 +89,12 @@ mkdir -p %{buildroot}/var/spacewalk
 mkdir -p %{buildroot}/%{_prefix}/share/rhn/config-defaults
 
 %pre server
+%if ! 0%{?suse_version} == 1110
```

Comment:
No RHEL 6 here? 

I wonder if we should exclude the complete server subpackage via macros.
Currently we would build an incomplete package. Maybe we do not build it at all?

Reply:
RHEL6 doesn't fail, only SLE11.

SLE11 fails because `-a` is not available at `usermod`, while that is not a problem at RHEL6.

Reply:
Then let's fix this first. We can discuss later about option B .